awards.md

```diff
@@ -19,19 +19,19 @@ An `epoch` is a fixed number of layers. Each Testnet epoch has XXX layers.
 
 So `Epoch 0` is the first XXX layers starting with `layer 0` and its last layer is `layer XXX-1`, and `Epoch 1` start in `Layer XXX` and ends with `Layer 2xXXX-1`.
 
-Your eligibility to contribute blocks to a specific layer is automatically determined by the Spacemesh protocol.
+Your eligibility to contribute blocks to a specific layer is determined by the Spacemesh protocol, and automatically executed by the App.
 
 The Spacemesh protocol guarantees that you will be eligible to submit at least one block in at least one layer in each and every epoch.
```

Comment:
This is not 100% accurate:
First, this is true as long as mining takes place.
Secondly, even after activating mining, one should wait a full epoch to produce an ATX, and then wait for the next epoch to start the actual mining.

Reply:
there is no way in spacemesh0.1 to run a full node w/o mining. If mining setup is not compete by the user then the full node is not started / restarted. so, from a user perspective there's either mining or he's running in wallet-only mode, at least for sm0.1. Maybe we should clarify this in this doc.

Reply:
The second point is more important.
For the first, there's a Pause mining button.

Reply:
I'm clarifying to Ilya - for 0.1 - the pause / play may just stop and resume mining.